frontend/src/pages/Policies.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { ReactElement } from 'react';
+import { Container, Link, Typography, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  h2: {
+    fontWeight: 600,
+    fontSize: '1.875em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '1.5em',
+    },
+  },
+  h3: {
+    fontWeight: 500,
+    fontSize: '1.5em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '1.25em',
+    },
+  },
+  h4: {
+    fontWeight: 500,
+    fontSize: '1.125em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '0.938em',
+    },
+  },
+  body: {
+    fontFamily: 'Work Sans, sans-serif',
+    color: 'black',
+    textAlign: 'left',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: '0.938em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '0.75em',
+    },
+    lineHeight: '20px',
+    marginBottom: '1em',
+  },
+  link: {
+    color: '#B94630',
+  },
+}));
+
+const Policies = (): ReactElement => {
+  const { h2, h3, h4, body, link } = useStyles();
+  return (
+    <Container maxWidth="lg">
```

Comment:
I think it would be better to add left and right padding (7% maybe?) to this container so the text isn't too close to the edges. Other than that, looks good!

Reply:
I left it like that so the text was aligned with the navbar text- let me know if you still think we should add padding!

Reply:
Oh I see. I don't think it matters that much, but I still think it would look nicer if we had some space on the sides.

Reply:
This is what 7% looks like
<img width="1319" alt="Screen Shot 2021-11-07 at 7 39 39 PM" src="https://user-images.githubusercontent.com/57203639/140668419-0cd6a960-20f8-4eec-8836-5bb2a9379417.png">

Reply:
I think I like it how it is now actually

Reply:
Alright I'll leave it as it is then.